src/App.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/extensions */
+import React, { useEffect } from 'react';
+import './App.css';
+import { useDispatch } from 'react-redux';
+import { Switch, Route, HashRouter as Router } from 'react-router-dom';
+import MainPage from './Components/MainPage';
+import Details from './Components/Details';
+import { loadCrypto } from './redux/reducers/cryptoReducers';
+
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loadCrypto());
```

Comment:
- [ ] The app still fails to start, it throws this error: `./src/App.js
Cannot find file: 'Details.js' does not match the corresponding name on disk: '.\src\Components\components'.`. Kindly fix it.

Reply:
✔